sunpy/physics/limb_darkening.py

```diff
@@ -0,0 +1,184 @@
+import numpy as np
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+
+
+def darklimb_u(wavelength):
+    """
+    Returns the constant, u, for a given wavelength for input into the limb
+    darkening function.
+
+    Parameters
+    ----------
+    wavelength : `~sunpy.map.GenericMap.wavelength`
+        The wavelength for which the limb darkening is to be calculated.
+
+    Notes
+    -----
+    Coefficients for 5th order polynomial fit to limb darkening
+    constant `u` was done on data from Astrophysical
+    Quantities by Allen.
+
+    Limitations
+    -----------
+    Only applies to wavelengths in the range 4000<lambda<15000 Angstrom
+
+    Reference
+    ---------
+    Cox, Arthur N., Allen's astrophysical quantities, 4th ed. Publisher:
+        New York: AIP Press; Springer, 2000. ISBN: 038798746
+
+    """
+
+    au = -8.9829751
+    bu = 0.0069093916
+    cu = -1.8144591e-6
+    du = 2.2540875e-10
+    eu = -1.3389747e-14
+    fu = 3.0453572e-19
+
+    a = np.array((au, bu, cu, du, eu, fu))
+
+    wavelength = wavelength.to_value()
+    wavelength_series = np.array([wavelength ** n for n in range(len(a))])
+
+    ul = np.sum(a * wavelength_series)
+
+    return ul
+
+
+def darklimb_v(wavelength):
+    """
+    Returns the constant, v, for a given wavelength for input into the limb
+    darkening function.
+
+    Parameters
+    ----------
+    wavelength : `~sunpy.map.GenericMap.wavelength`
+        The wavelength for which the limb darkening is to be calculated.
+
+    Notes
+    -----
+    Coefficients for 5th order polynomial fit to limb darkening
+    constant `v` was done on data from Astrophysical
+    Quantities by Allen.
+
+    Limitations
+    -----------
+    Only applies to wavelengths in the range 4000<lambda<15000 Angstrom
+
+    Reference
+    ---------
+    Cox, Arthur N., Allen's astrophysical quantities, 4th ed. Publisher:
+        New York: AIP Press; Springer, 2000. ISBN: 038798746
+
+    """
+
+    av = 9.2891180
+    bv = -0.0062212632
+    cv = 1.5788029e-6
+    dv = -1.9359644e-10
+    ev = 1.1444469e-14
+    fv = -2.599494e-19
+
+    a = np.array((av, bv, cv, dv, ev, fv))
+
+    wavelength = wavelength.to_value()
+    wavelength_series = np.array([wavelength ** n for n in range(len(a))])
+
+    vl = np.sum(a * wavelength_series)
+
+    return vl
+
+
+def get_limb_dark(sun_map):
```

Comment:
I never did like the name of this but I don't have any other ideas.

Reply:
I think if we make all these private we need to worry less.